src/map/Map.js

```diff
@@ -519,6 +519,7 @@ L.Map = L.Evented.extend({
 		this.createPane('shadowPane');
 		this.createPane('overlayPane');
 		this.createPane('markerPane');
+		this.createPane('labelPane');

```

Comment:
This is the situation I wanted to prevent:

![screenshot from 2015-10-18 09 32 41](https://cloud.githubusercontent.com/assets/146023/10563097/39fbc5d2-757b-11e5-8831-1a5e271f565c.png)

I think we want the labels always above the markers and vectors BUT always below the popups, right?

Dedicated pane seemed to me the strongest way to achieve that, other option is to play with z-index inside the popup pane.